src/endpoints/Models.jl

```diff
@@ -0,0 +1,22 @@
+# http://juliastats.github.io/StatsBase.jl/stable/weights.html
+mutable struct Packed_AliasingScalarSampler
+  samples::Vector{Float64}
+  weights::Vector{Float64}
+end
+
+## avert your eyes, C++ below
```

Comment:
Remove C++ excerpt below :)

Reply:
it might be good to leave an example here for people to see.